govuk_chat_evaluation/rag_answers/custom_deepeval/metrics/factual_correctness_completeness/factual_correctness_completeness.py

```diff
@@ -19,7 +20,12 @@
 import logging
 
 
-class FactualCorrectnessMetric(BaseMetric):
+class Mode(str, Enum):
+    CORRECTNESS = "correctness"
+    COMPLETENESS = "completeness"
```

Comment:
I think the value of these don't matter so we can use auto from enum to indicate it doesn't matter:


```suggestion
class Mode(Enum):
    CORRECTNESS = auto()
    COMPLETENESS = auto()
```

Reply:
btw only learnt about this feature this morning so could be wrong

Reply:
Interesting, I did think the way we were defining enums looked a bit redundant, duplicating the name as the value, without a specific need to do so.